### DISCUSSION TITLE
A Way to Flag a Repo to be Ignored

### DISCUSSION THREAD

**Discussion by christian-byrne** [Comfy-Org/ComfyUI-Manager#840] [General]
What would be the recommended way to have a repo you create be ignored and not added to the custom node list automatically? Could not find anything in the readme or discusssions/issues. Thanks.

**Comment by ltdrdata** [comment]
Come to think of it, there was nothing about that part.
I'm thinking maybe we could put a file named `.ignore-comfyui-manager` in the root directory.

**Reply by christian-byrne** [reply]
That's a great idea, seems like the best solution to me.

**Comment by christian-byrne** [comment]
@ltdrdata any update on this? 

**Context**: this [custom node security scanner](https://github.com/christian-byrne/custom-nodes-security-scan) is soon going to clone recently made custom nodes and run them in remote firejail sandbox, logging filesystem access violations and network connection, then doing post-execution static analysis. To test the program and setup baseline false-positive rates, I will be creating custom nodes that contain malware. 

Also, if you have any advice on the security scanner, I would be grateful for it. Thanks.

**Reply by ltdrdata** [reply]
Currently, a major improvement effort is underway for the node management system based on comfyregistry. Other tasks will proceed after this is completed.

**Reply by christian-byrne** [reply]
No problem. Can I have a timetable?

**Reply by ltdrdata** [reply]
It is currently entering alpha testing. After that, beta testing will be conducted for about two weeks, and it is expected to be released sometime after that.

**Reply by christian-byrne** [reply]
That works for me. Hope everything goes well with the release!
